components/collective-page/sections/Contribute.js

```diff
@@ -209,7 +223,9 @@ class SectionContribute extends React.PureComponent {
     const isProject = collective.type === CollectiveType.PROJECT;
     const isFund = collective.type === CollectiveType.FUND;
     const hasCustomContribution = !collective.settings?.disableCustomContributions;
-    const hasContribute = isAdmin || (collective.isActive && (sortedTiers.length || hasCustomContribution));
+    const hasCryptoContribution = !get(collective, 'settings.disableCryptoContributions', true);
```

Comment:
Why not look at the host setting here like on line 168?

Reply:
Yes, we should make it same, I've corrected that. Thanks. 👍🏽